dicom_standard/postprocess_merge_duplicate_nodes.py

```diff
@@ -0,0 +1,61 @@
+'''
+Merge nodes that have identical paths by adding conditional statments to the attribute descriptions.
+'''
+import re
+import sys
+from collections import Counter
+
+from bs4 import BeautifulSoup
+
+from dicom_standard import parse_lib as pl
+
+DUPLICATE_PATH_EXCEPTIONS = ['rt-segment-annotation:00700084']
+
+
+def get_description_text(raw_description):
+    description = BeautifulSoup(raw_description, "html.parser")
+    return description.getText()
+
+
+def add_conditional_to_description(node):
+    conditional = node.get('conditional')
+    assert conditional is not None, f'Duplicate attribute (path: {node["path"]}) has no conditional statement.'
+    conditional = re.sub(r'\.$', ':', conditional)
+    formatted_conditional = f'<p style="font-weight: bold">{conditional[0].upper()}{conditional[1:]}</p>'
+    node['description'] = formatted_conditional + node['description']
+
+
+def merge_duplicate_nodes(module_attr_list):
```

Comment:
I think this function is sufficiently complex that it would help to have some unit tests covering it, showing how the function works with simple mock data. That would also serve as a form of documentation, too.